crates/bevy_render/src/view/mod.rs

```diff
@@ -90,26 +91,141 @@ pub struct ViewUniformOffset {
     pub offset: u32,
 }
 
+pub enum ViewMainTexture {
+    Hdr {
+        hdr_texture: TextureView,
+        sampled_hdr_texture: Option<TextureView>,
+
+        ldr_texture: TextureView,
+    },
+    NoHdr {
```

Comment:
Sdr perhaps? NoHdr doesn't feel right. And I think SDR is what a non-HDR (ha!) display supports since forever. I think LDR is something else but I'm not certain... And if we choose SDR as being more correct than LDR, then using SDR everywhere for the labels and variable names and such.